.github/workflows/container.yaml

```
@@ -0,0 +1,92 @@
+name: Container Image
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to build (e.g., 1.42.4)'
+        required: true
+        type: string
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
```

Have you tested this workflow? I believe you'll need to add QEMU for running `docker build` for ARM64 (_**NOTE:** This can technically be avoided by modifying the `Dockerfile`_).

```suggestion
    - name: 'Set up QEMU'
      uses: docker/setup-qemu-action@v3
      with:
        platforms: arm64

    - name: Set up Docker Buildx
      uses: docker/setup-buildx-action@v3
```

The other option is using native GH runner for ARM64, but that adds complexity that isn't worth it for a simple image build.